Templates/MonoGame.Templates.CSharp/content/MonoGame.Application.Android.CSharp/MGNamespace.csproj

```diff
@@ -1,6 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>net6.0-android</TargetFramework>
+    <RollForward>Major</RollForward>
```

Comment:
Post merge regrets, iOS and Android aren't supposed to have it, there is no point for them 😅